### PR TITLE
Adding a few return types

### PR DESCRIPTION
**Pull request by kevmoo**
Would love feedback about doing abstract classes in AtScript

### CODE REVIEW

File: modules/angular2/src/core/application.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ export var appElementToken = new OpaqueToken('AppElement');
 export var appComponentAnnotatedTypeToken = new OpaqueToken('AppComponentAnnotatedType');
 export var appDocumentToken = new OpaqueToken('AppDocument');
 
-function _injectorBindings(appComponentType) {
+function _injectorBindings(appComponentType): List<Binding> {
```
Comment (mhevery) [inline modules/angular2/src/core/application.js]: I think you forgot to import `Binding` it breaks the CI

**Comment by mhevery** [comment]
Fix the broken build, LGTM otherwise

**Comment by kevmoo** [comment]
PTAL

**Comment by kevmoo** [comment]
I need someone to pull this in. I don't have commit access.

**Comment by kevmoo** [comment]
Waiting on rebase

**Comment by kevmoo** [comment]
@mhevery @yjbanov Please merge this for me – or give me a commit bit :smile:

**Comment by vicb** [comment]
3519714f64eb0c316394f3290a01e86aeb18aaca

**Comment by mhevery** [comment]
@kevmoo the `!merge` label is for us so that we know what is ready to merge. We periodically go and mass merge them. No work for you to do.

**Pull request closed without merging**